vendor/github.com/vishvananda/netlink/xfrm_policy_linux.go

```diff
@@ -248,6 +248,7 @@ func parseXfrmPolicy(m []byte, family int) (*XfrmPolicy, error) {
 				resTmpl.Mode = Mode(tmpl.Mode)
 				resTmpl.Spi = int(nl.Swap32(tmpl.XfrmId.Spi))
 				resTmpl.Reqid = int(tmpl.Reqid)
+				resTmpl.Optional = int(tmpl.Optional)
```

Comment:
This is the only practical difference from the encryption side. Basically when deserializing kernel objects, we will read the new field back out. I don't think we were relying on this side of things. /cc @jrfastab